src/__tests__/pick.test.ts

```diff
@@ -75,13 +75,13 @@ describe('pick', () => {
     });
     it('picks one field and one child field and does not validate if field is omitted', () => {
         const ChildFixture = pick(Child, 'ChildFixture', ['baz']);
-        const ParentBase = pick(Parent, 'ParentBase', ['foo', 'child']);
+        const ParentBase = pick(Parent, 'ParentBase', ['foo'], ['child']);
 
         class ParentFixture extends ParentBase {
             @IsNested({
                 type: ChildFixture,
             })
-            child!: Child; // THIS TYPE IS WRONG
+            child!: typeof ChildFixture;
```

Comment:
The previous test case here included `child` in the `pick()` expression, which caused the type of `child` in `ParentBase` to be the _old_ `Child` and not our derived `ChildFixture`.

With this change, `child` is no longer included in the return type of `ParentBase`, so our extension in `ParentFixture` "wins" the type definition battle.